plugins/inputs/ceph/README.md

```diff
@@ -46,15 +62,78 @@ Would be parsed into the following metrics, all of which would be tagged with co
 
   ## suffix used to identify socket files
   socket_suffix = "asok"
+
+  ## Ceph user to authenticate as
+  ceph_user = "client.admin"
```

Comment:
why was the ceph user not necessary in the previous plugin? and do we need to add a password configuration? or does the user need to do something to setup their admin credentials correctly?